include/sdf/Param.hh

```diff
@@ -326,6 +326,15 @@ namespace sdf
     public: bool SetParentElement(ElementPtr _parentElement,
                                   sdf::Errors &_errors);
 
+    /// \brief Set the parent Element of this Param without reparsing.
+    /// This is meant for internal consumption when cloning elements.
+    /// \param[in] _parentElement Pointer to new parent Element. A nullptr can
+    /// be provided to remove the current parent Element.
+    /// \param[out] _errors Vector of errors.
```

Comment:
This function doesn't have `_errors`, so remove?

Reply:
Fixed in https://github.com/gazebosim/sdformat/pull/1484/commits/d700b3557e8b67200484bf727fdb70555c66f587